finetune/util/chunk_long_seq.py

```diff
@@ -0,0 +1,148 @@
+
+
+from finetune.base import BaseModel, PredictMode
+
+def process_long_sequence(X, task, probas=False)
+    tasks = ['sequence_labeling', 'classification', 'regression']
+    assert task in tasks, 'invalid task for processing long sequences'
+    chunk_size = self.config.max_length - 2
+    step_size = chunk_size // 3
+    if task == 'sequence_labeling':
```

Comment:
Same as before -- I think we can call something like _format_for_inference or _format_for_encoding here rather than using a try except.